pkg/configurator/crd_methods.go

```diff
@@ -17,19 +17,19 @@ const (
 // The functions in this file implement the configurator.Configurator interface
```

Comment:
I don't see why the filename needs to be changed. Could you keep this as `methods.go`?

Reply:
I initially thought this would be cleaner for code review.

But now seems keep methods.go makes more sense.

Will update.